vim-engine/src/main/kotlin/com/maddyhome/idea/vim/mark/VimMarkGroup.kt

```diff
@@ -10,7 +10,10 @@ package com.maddyhome.idea.vim.mark
 
 import com.maddyhome.idea.vim.api.VimEditor
 import com.maddyhome.idea.vim.common.TextRange
+import org.jetbrains.annotations.ApiStatus
 
+@Deprecated("Please use VimMarkService instead")
+@ApiStatus.ScheduledForRemoval(inVersion = "2.3")
 interface VimMarkGroup {
```

Comment:
You can execute `Find external usages` action on every of the method. If there are no local and no external usages, the method is safe to delete.